azure-resources/VirtualMachineImages/imageTemplates/recommendations.yaml

```diff
@@ -24,14 +24,14 @@
   recommendationResourceType: Microsoft.VirtualMachineImages/imageTemplates
   recommendationMetadataState: Active
   longDescription: |
-    The Azure Image Builder service, used for deploying Image Templates, lacks availability zones support. By replicating Image Templates to a secondary, preferably paired, region, quick recovery from a region failure is enabled, ensuring continuous virtual machine deployment from these templates.
+    The Azure Image Builder service lacks availability zones support. Replicating Image Templates to a secondary region will enable the build of new images in secondary region.
```

Comment:
This one doesn't make sense to me. 

First sentence mentioned lack of AZ support and second sentence refers to secondary region. 

Might make sense to have something closer to:

The Azure Image Builder Service is a regional service. Replicate Image Templates to a secondary region to enable build of new images in secondary region if/when required.

Though guid b14ee8ed-7d27-447b-b6fb-6472cb5f4b75 says something similar as well, we may remove this one?